### PR TITLE
Doc: How to apply autocompletion in bash

### PR DESCRIPTION
**Pull request by Linutux**
Fixes https://github.com/kubernetes/kubernetes/issues/111831

### CODE REVIEW

File: content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 1]
```diff
@@ -51,3 +51,10 @@ bash-completion sources all completion scripts in `/etc/bash_completion.d`.
 {{< /note >}}
 
 Both approaches are equivalent. After reloading your shell, kubectl autocompletion should be working.
+
+To enable autocompletion in current session of shell, source the .bashrc file:
+
+```bash
+source ~/.bashrc
+```bash
```
Comment (tengqm) [inline content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:59]:
````suggestion
```
````

File: content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 2]
```diff
@@ -51,3 +51,10 @@ bash-completion sources all completion scripts in `/etc/bash_completion.d`.
 {{< /note >}}
 
 Both approaches are equivalent. After reloading your shell, kubectl autocompletion should be working.
+
+To enable autocompletion in current session of shell, source the .bashrc file:
```
Comment (sftim) [inline content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:55]:
```suggestion
To enable autocompletion for your current shell sessions, source your `.bashrc` file:
```

File: content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 3]
```diff
@@ -51,3 +51,10 @@ bash-completion sources all completion scripts in `/etc/bash_completion.d`.
...
+
+```bash
+source ~/.bashrc
```
Comment (sftim) [inline content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:58]:
```suggestion
# run this in each local shell session you have open
source ~/.bashrc
```

**Review by sftim — COMMENTED** [review]
This advice assumes that it's safe to source `~/.bashrc`.

I would instead run `exec bash` to replace the current shell with a fresh session. Do you prefer that @Linutux?

File: content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md (hunk 2 above)
Comment (sftim) [inline content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:55]: Does this work if the user picked the “System” option, editing `/etc/bash_completion.d/kubectl`?

**Comment by sftim** [comment]
I'm minded not to approve this as-is because I have spotted what looks like shortcomings, and I can recommend a different approach that works for more end users.

**Comment by mehabhalodiya** [comment]
/lgtm cancel
@Linutux Please make the changes suggested by Tim.

**Comment by Sea-n** [comment]
Hey @Linutux,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by Linutux** [comment]
> Hey @Linutux,
> 
> It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?
I'm so sorry for not responding. I will address the issues by Monday.

**Comment by Sea-n** [comment]
Hey @Linutux,

Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.
(Note that please don't push anything while PR closed, or you won't be able to reopen it.)

**Pull request closed without merging**